test/app/dftb+/hybrid/periodic/Acene-kpts-matrix-restart-I/dftb_in.hsd

```diff
@@ -0,0 +1,63 @@
+Geometry = GenFormat {
```

Comment:
Am I misunderstanding, or is this file not removed in testrun.sh before any calculation starts?